mocha_test/forever_test.js

```diff
@@ -0,0 +1,44 @@
+var async = require('../lib/async');
```

Comment:
Not a big fan of the suffixed file names. Why not just run tests on everything in `mocha_test/*`